packages/ses/src/module-load.js

```diff
@@ -92,16 +92,18 @@ const loadWithoutErrorAnnotation = async (
     aliasNamespace = moduleMapHook(moduleSpecifier);
   }
   if (typeof aliasNamespace === 'string') {
-    throw new TypeError(
-      `Cannot map module ${q(moduleSpecifier)} to ${q(
+    // TODO Prefer TypeError if we can ask assert to do that.
```

Comment:
TODO: Ticket for adding an argument to `fail` with the constructor for the error type.

Reply:
Even better. Since that TODO was written I did that. But I hadn't noticed the opportunity to fix this line to use it. Done.